ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/PaginatingAsyncStream.scala

```diff
@@ -9,8 +9,9 @@ import akka.stream.scaladsl.Source
 import scala.concurrent.{ExecutionContext, Future}
 
 object PaginatingAsyncStream {
-  def apply[T](pageSize: Int, executionContext: ExecutionContext)(
-      queryPage: Long => Future[List[T]]): Source[T, NotUsed] = {
+  def apply[T](pageSize: Int)(queryPage: Long => Future[List[T]])(
```

Comment:
Similar to @stefanobaghino-da's comment in `JdbcLedgerDao`, please change back to use the execution context explicitly.

Reply:
Done.